### DISCUSSION TITLE
Blogeintrag Woche 07 (Sem4) - CI/CD

### DISCUSSION THREAD

**Discussion by DossohMael** [mausio/corgi-shop-doc#17] [Blogs]
Zur Verbesserung der Codequalität wollen wir SonarQube in unseren CI/CD-Prozess integrieren. Dieses Tool analysiert Code und stellt sicher, dass er den höchsten Standards entspricht. Die gesamte CI/CD-Pipeline soll auf GitHub stattfinden, damit sie nahtlos in unseren bestehenden Entwicklungsworkflow integriert werden kann.
Bereits haben wir eine grundlegende Infrastruktur mit Docker Compose eingerichtet, die eine PostgreSQL-Datenbank in einem Docker-Container bereitstellt. 

Bedingt einer Klausur kam es bei uns diese Woche nur zur Planung der nächsten Arbeitsschritte.  
Die weitere Implementierung folgt in der nächsten Woche.

**Comment by JannisFuchs** [comment]
SonarQube hört sich nach einer guten Möglichkeit an

**Comment by gitricko** [comment]
Vielleicht möchten Sie Sonarless ausprobieren, um eine einfache Möglichkeit zum Einrichten des Sonarqube-Scannens und der Github-Aktionsintegration zu erhalten

https://github.com/gitricko/sonarless
